packages/agent/src/factory/createAutoBeContext.ts

```diff
@@ -205,10 +205,25 @@ export const createAutoBeContext = <Model extends ILlmSchema.Model>(props: {
         if (closure) closure(agent);
 
         // DO CONVERSATE
-        const message: string =
+        const message:
```

Comment:
Rollback to the text type please.

This is not conversation from facade or user, but internal workflow agent.

We never plan to provide an image from the internal workflow agent.